global/scripts/debug.shlib

```diff
@@ -0,0 +1,262 @@
+#!/bin/echo This script should not run directly:
+
+# Description:
+#
+#   Function for printing debugging information on caught errors.
+#   It makes use of error trapping. To reset the trap, add
+#   end of the script using "trap - ERR".
```

Comment:
FWIW, I don't understand this comment. Add what to the end of what script?

Reply:
The new way to disable it is to run `debug_disable_trap`.  The comment should be rewritten.

Reply:
Removed the confusing part.